### PR TITLE
Add complete moduleinfo.json example

### PR DESCRIPTION
**Pull request by grandeljay**
Whenever I start building a new module, I keep looking for a template I can copy & paste to get going. This PR adds a complete `moduleinfo.json` example and includes some minor fixes.

It would also be nice if you would offer a `moduleinfo.json` generator. Basically just a form where I can enter some relevant info and then just use whatever it spits out.

### CODE REVIEW

File: examples/full-example.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "shortDescription": "",
+    "description": "",
+
+    "developer": "Robin Wieschendorf",
+    "developerWebsite": "https://robinwieschendorf.de",
+    "website": "https://github.com/RobinTheHood/modified-std-module",
+
+    "category": "productivity",
+    "price": "29.90",
+    "tags": "verstand, shipping, paket",
```
Comment (RobinTheHood) [inline examples/full-example.json]: I hope you have a lot of "verstand" 😉

File: examples/full-example.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+    "name": "My first Module",
+    "archiveName": "mycompany/my-first-module",
+    "sourceDir": "new_files",
```
Comment (RobinTheHood) [inline examples/full-example.json]: Maybe we should use "src" already, even if "new_files" is still often used. But I think the default should be "src".

File: examples/full-example.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        "2.0.6.0",
+        "2.0.7.0",
+        "2.0.7.1",
+        "2.0.7.2"
+    ],
+    "php": {
+        "version": "^7.4 || ^8.0"
+    },
+    "mmlc": {
+        "version": "^1.20.0"
```
Comment (RobinTheHood) [inline examples/full-example.json]: I think it's good if we use "^1.21.0" here.

**Review by RobinTheHood — CHANGES_REQUESTED** [review]
Thanks for your PR, I added a few comments.

File: examples/full-example.json (hunk 1 above)
Comment (grandeljay) [reply examples/full-example.json]: I figured it was a typo but copied it from the documentation for completeness :D

https://github.com/RobinTheHood/module-loader.de/blob/1a60419342b082f800e90b4f361f1162110859a6/public/docs/moduleinfo.php#L224

Comment (RobinTheHood) [reply examples/full-example.json]: Oh shit. Maybe you can fix it for me 😅 Thx

File: examples/full-example.json (hunk 2 above)
Comment (grandeljay) [reply examples/full-example.json]: Great! I also prefer `src`.

File: public/docs/moduleinfo.php [hunk 4]
```diff
@@ -50,7 +51,17 @@
...
+                    </section>
+
+                    <section>
+                        <h3 id="example">
+                            Beispiel
+                            <a href="#example" class="anchor">#</a>
+                        </h3>
+
+                        <p>Eine Vollständige <code>moduleinfo.json</code> könnte wie folgt aussehen.</p>
+                        <code class="block"><?= file_get_contents('../../examples/full-example.json'); ?></code>
```
Comment (RobinTheHood) [inline public/docs/moduleinfo.php]: Get the latest commit and look in docs/tutorial.php. 
Look vor `<?= loadExample2('/tutorial_010.json', 'json'); ?>`. You should also use `<= loadExample2(FILENAME, FILETYPE); =>`

**Review by RobinTheHood — APPROVED** [review]

**Review by RobinTheHood — APPROVED** [review]

**Pull request merged**